environments/openshift/manifests/observatorium-template.yaml

```diff
@@ -878,6 +878,218 @@ objects:
         - name: telemeter-server-tls
           secret:
             secretName: telemeter-server-shared
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRole
+  metadata:
+    name: prometheus-ams-${NAMESPACE}
+  rules:
+  - nonResourceURLs:
+    - /metrics
+    verbs:
+    - get
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRoleBinding
+  metadata:
+    name: prometheus-ams-${NAMESPACE}
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: prometheus-ams-${NAMESPACE}
+  subjects:
+  - kind: ServiceAccount
+    name: prometheus-ams
+    namespace: ${NAMESPACE}
+- apiVersion: v1
+  data:
+    nginx.conf: |
+      daemon off;
+      worker_processes 1;
+      error_log /dev/stderr;
+      pid /tmp/nginx.pid;
+
+      events {
+          worker_connections 1024;
+      }
+
+      http {
+        log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                          '$status $body_bytes_sent "$http_referer" '
+                          '"$http_user_agent" "$http_x_forwarded_for"';
+
+        upstream app {
+          server http://thanos-receive.${NAMESPACE}.svc.cluster.local:10902/api/v1/receive;
+          keepalive 64;
+        }
+
+        server {
+          listen *:8080;
+          server_name _;
+          access_log  /dev/stdout  main;
+          error_log /dev/stderr;
+
+          location / {
+            proxy_set_header THANOS-TENANT FB870BF3-9F3A-44FF-9BF7-D7A047A52F43;
+            proxy_pass http://app;
+          }
+        }
+      }
+  kind: ConfigMap
+  metadata:
+    labels:
+      prometheus: ams
+    name: remote-write-proxy-config
+    namespace: ${NAMESPACE}
+- apiVersion: monitoring.coreos.com/v1
+  kind: Prometheus
+  metadata:
+    labels:
+      prometheus: ams
+    name: ams
+    namespace: ${NAMESPACE}
+  spec:
+    baseImage: ${PROMETHEUS_IMAGE}
+    containers:
+    - args: []
+      image: quay.io/app-sre/observatorium-receive-proxy:latest
```

Comment:
Can we parameterize the image tag instead of using latest? Its okay to start with a hardcoded version, works better than `latest`